install-dev/data/xml/tab.xml

```diff
@@ -5,450 +5,697 @@
+                    <wording>Customer Service</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                 <tab id="Order_Messages" id_parent="Customer_Service" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminOrderMessage</class_name>
+                    <wording>Order Messages</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
                 <tab id="Merchandise_Returns" id_parent="Customer_Service" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminReturn</class_name>
+                    <wording>Merchandise Returns</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
 
             <!--STATISTICS-->
             <tab id="Stats" id_parent="SELL" active="1" enabled="1" hide_host_mode="0" icon="assessment">
                 <class_name>AdminStats</class_name>
+                <wording>Stats</wording>
+                <wording_domain>Admin.Navigation.Menu</wording_domain>
             </tab>
 
             <!--ADVANCED STOCK MANAGEMENT-->
             <tab id="Advanced_stock_management" id_parent="SELL" active="1" enabled="1" hide_host_mode="0" icon="store">
                 <class_name>AdminStock</class_name>
+                <wording/>
+                <wording_domain/>
             </tab>
 
                 <tab id="Warehouses" id_parent="Advanced_stock_management" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminWarehouses</class_name>
+                    <wording>Warehouses</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
-                <tab id="Stock_Management" id_parent="Advanced_stock_management" active="1" enabled="1" hide_host_mode="0" icon="">
+                <tab id="Stock_Management_Parent" id_parent="Advanced_stock_management" active="1" enabled="1" hide_host_mode="0" icon="">
                     <class_name>AdminParentStockManagement</class_name>
+                    <wording>Stock Management</wording>
+                    <wording_domain>Admin.Navigation.Menu</wording_domain>
                 </tab>
 
-                    <tab id="Stock_Management" id_parent="Stock_Management" active="1" enabled="1" hide_host_mode="0" icon="">
+                    <tab id="Stock_Management" id_parent="Stock_Management_Parent" active="1" enabled="1" hide_host_mode="0" icon="">
                         <class_name>AdminStockManagement</class_name>
+                        <wording>Stock Management</wording>
+                        <wording_domain>Admin.Navigation.Menu</wording_domain>
                     </tab>
                     <tab id="Stock_Movement" id_parent="Stock_Management" active="1" enabled="1" hide_host_mode="0" icon="">
```

Comment:
```suggestion
                    <tab id="Stock_Movement" id_parent="Stock_Management_Parent" active="1" enabled="1" hide_host_mode="0" icon="">
```
?

Reply:
And for others

Reply:
Again, this is shadowy terrain that should definitely be explored, but not in this PR!